passkit_server/lib/src/passkit_server.dart

```diff
@@ -0,0 +1,212 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:passkit_server/src/passkit_backend.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf_router/shelf_router.dart';
+
+extension PasskitServerExtension on Router {
+  void addPassKitServer(PassKitBackend backend) {
+    post(
+      '/v1/devices/<deviceID>/registrations/<passTypeID>/<serial>',
+      setupNotifications(backend),
+    );
+    get(
+      '/v1/devices/<deviceID>/registrations/<typeID>',
+      getListOfUpdatablePasses(backend),
+    );
+    delete(
+      '/v1/devices/<deviceID>/registrations/<passTypeID>/<serial>',
+      stopNotifications(backend),
+    );
+    get('/v1/passes/<identifier>/<serial>', getLatestVersion(backend));
+    post('/v1/log', logMessages(backend));
+  }
+}
```

Comment:
This should probably be a shelf routing middleware. That ensures better compatibility with dart_frog. For shelf_router compatibility should still be maintained though, but I guess that's easy enough to do and requires just some documentation or example code. That way, this package also doesn't depend on the shelf_router package itself.

References:
- https://pub.dev/documentation/dart_frog/latest/dart_frog/fromShelfMiddleware.html
- https://pub.dev/packages/shelf#handlers-and-middleware